player/lua/stats.lua

```diff
@@ -721,114 +736,196 @@ local function pq_eotf(x)
-            append(s, hdr["max-fall"], {prefix="maxFALL:", suffix=" cd/m²", nl=""})
+            append(s, hdr["max-fall"], {prefix="MaxFALL:", suffix=" cd/m²", nl="",
+                                        indent=indent})
         end
     end
 
+    indent = o.prefix_sep .. o.prefix_sep
+
     if hdr["scene-max-r"] or hdr["scene-max-g"] or
        hdr["scene-max-b"] or hdr["scene-avg"] then
         append(s, "", {prefix="HDR10+:"})
         append(s, format("%.1f / %.1f / %.1f", hdr["scene-max-r"] or 0,
                          hdr["scene-max-g"] or 0, hdr["scene-max-b"] or 0),
-               {prefix="MaxRGB:", suffix=" cd/m²", nl=""})
+               {prefix="MaxRGB:", suffix=" cd/m²", nl="", indent=""})
         append(s, format("%.1f", hdr["scene-avg"] or 0),
-               {prefix="Avg:", suffix=" cd/m²", nl=""})
+               {prefix="Avg:", suffix=" cd/m²", nl="", indent=indent})
     end
 
     if hdr["max-pq-y"] and hdr["avg-pq-y"] then
         append(s, "", {prefix="PQ(Y):"})
         append(s, format("%.2f cd/m² (%.2f%% PQ)", pq_eotf(hdr["max-pq-y"]),
-                         hdr["max-pq-y"] * 100), {prefix="Max:", nl=""})
+                         hdr["max-pq-y"] * 100), {prefix="Max:", nl="",
+                         indent=""})
         append(s, format("%.2f cd/m² (%.2f%% PQ)", pq_eotf(hdr["avg-pq-y"]),
-                         hdr["avg-pq-y"] * 100), {prefix="Avg:", nl=""})
+                         hdr["avg-pq-y"] * 100), {prefix="Avg:", nl="",
+                         indent=indent})
     end
 end
 
 
-local function add_video(s)
-    local r = mp.get_property_native("video-params")
-    local ro = mp.get_property_native("video-out-params")
-    -- in case of e.g. lavfi-complex there can be no input video, only output
-    if not r then
-        r = ro
-    end
+local function append_img_params(s, r, ro)
     if not r then
         return
     end
 
-    local osd_dims = mp.get_property_native("osd-dimensions")
-    local scaled_width = osd_dims["w"] - osd_dims["ml"] - osd_dims["mr"]
-    local scaled_height = osd_dims["h"] - osd_dims["mt"] - osd_dims["mb"]
+    append_resolution(s, r, "Resolution:", "w", "h", true)
+    if ro and (r["w"] ~= ro["dw"] or r["h"] ~= ro["dh"]) then
+        if ro["crop-w"] and (crop_noop(r["w"], r["h"], ro) or crop_equal(r, ro)) then
+            ro["crop-w"] = nil
+        end
+        append_resolution(s, ro, "Output Resolution:", "dw", "dh")
+    end
 
-    append(s, "", {prefix=o.nl .. o.nl .. "Video:", nl="", indent=""})
-    if append_property(s, "video-codec", {prefix_sep="", nl="", indent=""}) then
-        append_property(s, "hwdec-current", {prefix="(hwdec:", nl="", indent=" ",
-                         no_prefix_markup=true, suffix=")"}, {no=true, [""]=true})
+    local indent = o.prefix_sep .. o.prefix_sep
+
+    local pixel_format = r["hw-pixelformat"] or r["pixelformat"]
+    append(s, pixel_format, {prefix="Pixel Format:"})
```

Comment:
didn't these used to get displayed alongside?

Reply:
Yes, it is explained in the commit msg. 

Basically mpv uses "fake" imgfmt for hwdec, which are used internal to track the format. But it is not real pixel format, it doesn't mean anything. For the fact that hwdec is enabled we display `hwdec-current`.